modules/rosa-creating-cluster.adoc

```diff
@@ -15,20 +15,22 @@ You have completed the installation prerequisites.
 
 .Procedure
 
-Enter the following command to create your cluster with the default cluster settings.
-
+. Enter the following command to create your cluster with the default cluster settings.
++
+[source, terminal]
+----
+$ rosa create cluster --cluster-name=rh-rosa-test-cluster1
+----
++
 [NOTE]
 ====
+Multi-availability zones (AZ) are recommended for ROSA production workloads. The default is a single availability zone. Use either `--help` for an example of how to set this option or use interactive mode to be prompted for this setting.
+
```

Comment:
The feedback from myself and @bergerhoffer didn't make it to this new PR

Reply:
What I meant was I incorporated your feedback into the content itself, but I did not copy over either of your actual comments. But here they are for your reference:

bergerhoffer: 

I think that "either" is throwing me off on the second sentence. I think you could just say "Use --help to set the option manually or use interactive mode to be prompted for this setting."

Also, I can't imagine that the --help flag actually sets the option manually. I would think that help might just give you an example of how to set the option manually. Should it be "Use --help for an example of how to set the option manually or use interactive mode to be prompted for this setting." ?

jeder: 

Agreed to wqhat @bergerhoffer said. Also s/Multi-availability/Multiple availability zones

and s/ROSA//


I also just updated the `describe cluster` example in step 2 with the one you gave me in your separate PR this morning..